pallets/pool-system/src/solution.rs

```diff
@@ -239,34 +234,21 @@ impl<Balance> EpochSolution<Balance> {
 			let mut acc_invest = Balance::zero();
 			let mut acc_redeem = Balance::zero();
 			tranches.combine_with_residual_top(solution, |tranche, solution| {
-				acc_invest = solution
-					.invest_fulfillment
-					.mul_floor(tranche.invest)
-					.checked_add(&acc_invest)
-					.ok_or(ArithmeticError::Overflow)?;
-
-				acc_redeem = solution
-					.invest_fulfillment
```

Comment:
I would want to confirm if this `invest_fullfillment` should be `redeem_fulfillment` instead. Seems like in others places where the accumulator of redeems is computed, the `reedem_fulfillment` is used. @offerijns @mustermeiszer ?

Reply:
Great catching this @lemunozm!